packages/flutter_tools/lib/src/debug_adapters/flutter_adapter.dart

```diff
@@ -382,7 +382,7 @@ class FlutterDebugAdapter extends FlutterBaseDebugAdapter {
   }
 
   /// Handles the app.start event from Flutter.
-  void _handleAppStart(Map<String, Object?> params) {
+  Future<void> _handleAppStart(Map<String, Object?> params) async {
```

Comment:
Why was this changed to be async?

Reply:
hmm, I'm not sure. Maybe I originally added some async code in here and didn't revert it properly (or maybe I was testing you 😄).

Removed, thanks!

Reply:
Whew, glad I passed the test